docs/current/guides/snippets/use-services/environment-variables/index.ts

```diff
@@ -0,0 +1,27 @@
+import Client, { connect } from "@dagger.io/dagger"
+
+type Container = ReturnType<Client["container"]>;
```

Comment:
Off-topic, but we need to do something about this. The client types need to be properly exported. Did you explicitly avoid importing from `dist`?
```typescript
import { Container } from "@dagger.io/dagger/dist/api/client.gen"
```

Reply:
I didn't manage to make it work

Reply:
Maybe you need to change the extension to `.mts`.

Reply:
> Maybe you need to change the extension to `.mts`. Assuming you're following the instructions in [Get Started](https://docs.dagger.io/sdk/nodejs/783645/get-started#step-2-create-a-dagger-client-in-nodejs).

I was, still an issue on my end. You've already made it working on your side ?

Reply:
Any idea @slumbering ?

Reply:
> Off-topic, but we need to do something about this. The client types need to be properly exported. 

Totally agree. I'm going to take a look for this example in particular + look for the proper way to export types.

Reply:
> Any idea @slumbering ? @crjm

Yep ! Just did a quick fix to export the type correctly.